crates/ruma-state-res/src/lib.rs

```diff
@@ -1,11 +1,11 @@
 use std::{
     cmp::Reverse,
-    collections::{BTreeMap, BTreeSet, BinaryHeap},
+    collections::{BinaryHeap, HashMap, HashSet},
     sync::Arc,
 };
 
 use itertools::Itertools;
-use maplit::btreeset;
+use maplit::{hashset};
```

Comment:
`cargo fmt` didnt catch this? :thinking:

Reply:
`cargo jonathan` did yay.